README.md

```diff
@@ -61,6 +61,12 @@ PUPPETEER_HEADLESS=false npx @puppeteer/replay recording.json # runs in headful
 PUPPETEER_HEADLESS=chrome npx @puppeteer/replay recording.json # runs in the new experimental headless mode.
 ```
 
+Set the `BASE_URL` environment variable to replace your base URL in your recordings. For example,
+
+```
+BASE_URL=localhost:3001 npx @puppeteer/replay recording.json # It will run your recording.json by replacing the base URLs of your URLs with the BASE_URL you provided
```

Comment:
Maybe here you want to add a bit more info as to which URLs. In this case the URLs of navigate steps. 

Set the `BASE_URL` environment variable to replace the base URL in the navigate steps of your recordings. For example,

# replaces the base URLs in the navigate steps